src/Operation/RemoteMergeOperation.php

```diff
@@ -143,6 +147,21 @@ public function pushToRemote()
         $this->gitHelper->pushToRemote($this->targetRemote, $target);
     }
 
+    public function rebase(bool $rebase = false)
+    {
+        $this->rebase = $rebase;
+        $this->guardSync = !$rebase;
```

Comment:
Also I'm not sure that this is the best way to do it, since `--rebase` is not passed through, this will set `guardSync` to true, which I'm not sure is the desired behaviour. If the intention is that this should always default to `true`, then the command help should state that, as well as setting the correct default values in the definition